ts/Data/Converters/CSVConverter.ts

```diff
@@ -636,12 +630,12 @@ namespace CSVConverter {
      * Options that are not compatible with ClassJSON
      */
     export interface SpecialOptions {
-        beforeParse?: DataBeforeParseCallbackFunction;
+        beforeParse?: BeforeParseCallbackFunction;
```

Comment:
Does it work when dataPolling is enabled?

Reply:
demo added